apps/aecore/test/aec_blocks_tests.erl

```diff
@@ -85,14 +85,21 @@ network_serialization_test_() ->
 
 validate_test_() ->
     {setup,
-     fun aec_test_utils:aec_keys_setup/0,
-     fun aec_test_utils:aec_keys_cleanup/1,
+     fun() ->
+             ok = meck:new(aec_chain, [passthrough]),
+             meck:expect(aec_chain, get_top_state, 0, {ok, aec_trees:new()}),
+             aec_test_utils:aec_keys_setup()
+     end,
```

Comment:
I don't think you need this anymore, since the signature check went away.

Reply:
Well we shall either `meck` signed coinbases in this module or use keys...

Reply:
Yes, but you don't need the mecking of the chain, I think. No big deal, though.

Reply:
Addressed